scripts/requirements-fixed.txt

```diff
@@ -7,23 +19,23 @@ astroid==2.4.2
 attrs==20.3.0
 Babel==2.9.0
 breathe==4.26.1
+cached-property==1.5.2
 canopen==1.2.0
 cbor==1.0.0
 certifi==2020.12.5
 cffi==1.14.4
 chardet==4.0.0
 click==7.1.2
-cmake==3.17.3
 cmsis-pack-manager==0.2.10
 colorama==0.4.4
 commonmark==0.9.1
-configobj==5.0.6
-Deprecated==1.2.10
 coverage==5.4
 cryptography==3.3.1
+Deprecated==1.2.11
 docopt==0.6.2
 docutils==0.16
 ecdsa==0.16.1
+editdistance==0.5.3
```

Comment:
`editdistance` is something new and currently this breaks installation on Windows in Azure CI.

Reply:
it came from requirements-extra.txt

Reply:
I don't know, this is the latest version and it looks like prebuilt wheels are only available up to python 3.7.
We could drop this or downgrade to python 3.7?

Reply:
can you just remove it from the windows image with sed/awk.

Reply:
@tejlmand says everything in extra is needed by users, 
why take it away from linux/mac users because it isn't in windows.

Reply:
I'll do so.

Reply:
interesting, I just ran `python -mpip install -r requirements-fixed.txt` on window with no errors.
Does the Azure do anything special ?

Reply:
>  don't know, this is the latest version and it looks like prebuilt wheels are only available up to python 3.7.

Guess this answer my question.

Reply:
Build passed on Windows by excluding `editdistance`.